portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,25 @@
 @WebServlet("/data")
```

Comment:
Is the guide telling you to tall this data? Otherwise I'd use a more descriptive name, as "randomName" for example, or "firstName".

Reply:
/data is a path that will redirect to something else in the future, so this will not be needed.